frameworks/digital-outcomes-and-specialists-5/questions/services/agileCoachLocations.yml

```diff
@@ -24,4 +24,4 @@ options:
 
 validations:
   - name: answer_required
-    message: 'You need to answer this question.'
+    message: Select the location or locations
```

Comment:
Can we make this shorter? The GOV.UK style guide seems to suggest [just using the plural](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#brackets)

(and all other 'X or Xs' in messages)

Reply:
It's based on [Design System guidance on errors for checkboxes](https://design-system.service.gov.uk/components/checkboxes/#error-messages).

I'm happy to change it but I'd want to see what Nora thinks first.